dhis-2/dhis-services/dhis-service-acl/src/main/java/org/hisp/dhis/security/acl/DefaultAclService.java

```diff
@@ -474,7 +478,7 @@ public boolean canManage( User user, IdentifiableObject object )
             return errorReports;
         }
 
-        if ( !AccessStringHelper.isValid( object.getPublicAccess() ) )
+        if ( !AccessStringHelper.isValid( object.getSharing().getPublicAccess() ) )
```

Comment:
Would it make sense to move this validation logic to Sharing? sharing.validate() could return a list of ErrorReport